tiatoolbox/tools/stainnorm/stain_normaliser.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+
+from tiatoolbox.utils.transforms import convert_OD2RGB, convert_RGB2OD
+from tiatoolbox.tools.stainnorm.stain_extraction.ruifrok_stain_extractor import (
+    RuifrokStainExtractor,
+)
+
+
+class StainNormaliser:
+    """Stain normalisation class
+
+    Attributes:
+        method (string): stain normalisation method to use
```

Comment:
attributes are not complete

Reply:
Can you define what you mean by attributes here? Class attributes or instance attributes?

Reply:
Please consider the example at this [link](https://github.com/TIA-Lab/tiatoolbox/blob/a826f658dd02671631d83cfc42676b2858548fb5/tiatoolbox/dataloader/wsireader.py#L35)

Reply:
self.target_means = means
self.target_stds = stds

target_means and target_stds will be class attributes?